MaterialLib/SolidModels/CreateLubby2.h

```diff
@@ -93,20 +94,79 @@ std::unique_ptr<Lubby2<DisplacementDim>> createLubby2(
     DBUG("Use '%s' as dependency parameter mvM.",
          dependency_parameter_mvM.name.c_str());
 
-    Lubby2MaterialProperties mp{
-        kelvin_shear_modulus,     maxwell_shear_modulus,
-        maxwell_bulk_modulus,     kelvin_viscosity,
-        maxwell_viscosity,        dependency_parameter_mK,
-        dependency_parameter_mvK, dependency_parameter_mvM};
+    auto const dependency_parameter_tref_name = config.getConfigParameterOptional<std::string>("dependency_parameter_tref");
```

Comment:
- clang-format
- parameter documentation

Reply:
@wenqing OK, will reformat and document...